dataactbroker/handlers/fileHandler.py

```diff
@@ -1664,12 +1664,17 @@ def list_submissions(page, limit, published, sort='modified', order='desc', is_f
     user_columns = [User.user_id, User.name, publishing_user.user_id.label('publishing_user_id'),
                     publishing_user.name.label('publishing_user_name')]
     view_columns = [submission_updated_view.submission_id, submission_updated_view.updated_at.label('updated_at')]
-    sub_query = sess.query(PublishHistory.submission_id, func.max(PublishHistory.created_at).label('published_date')).\
-        group_by(PublishHistory.submission_id).\
-        subquery()
+    max_pub = sess.query(PublishHistory.submission_id, func.max(PublishHistory.created_at).label('max_date')).\
+        group_by(PublishHistory.submission_id).cte('max_pub')
+    max_cert = sess.query(CertifyHistory.submission_id, func.max(CertifyHistory.created_at).label('max_date')). \
+        group_by(CertifyHistory.submission_id).cte('max_cert')
+    pub_query = sess.query(max_pub.c.submission_id,
+                           case([(func.coalesce(max_cert.c.max_date, '1/1/1973') > max_pub.c.max_date,
```

Comment:
Is the new epoch date 1/1/1973? I always thought it was 1970 or 71?

Reply:
It is indeed, although this doesn't REALLY matter as long as the date we're using is after like... 2016 in this case, but I can update it to the epoch